src/components/global/Footer.vue

```diff
@@ -16,12 +17,23 @@
 </template>
 
 <script lang="ts">
-import Vue from 'vue';
+import { Component, Vue } from 'vue-property-decorator';
 
-export default Vue.extend({
+@Component({
 	name: 'Footer',
-	props: {},
-});
+	components: {},
+})
+export default class Footer extends Vue {
+	private version: string = '';
+
+	private async created(): Promise<void> {
+		fetch('https://api.github.com/repos/diging/vogon-web/releases/latest')
```

Comment:
Why use fetch when we have axios? I did not realize that you were using fetch the first time around, I just noticed the two thens. Please change this to axios and get rid of the two thens since axios transforms the json automatically.